build.gradle

```diff
@@ -283,8 +272,8 @@ task runProviderPactVerification(type: Test) {
 }
 
 task fortifyScan(type: JavaExec)  {
-  mainClass.set("uk.gov.hmcts.fortifyclient.FortifyClientMainApp")
-  classpath += sourceSets.fortifyTest.runtimeClasspath
+  mainClass = "uk.gov.hmcts.fortifyclient.FortifyClientMainApp"
+  classpath += sourceSets.test.runtimeClasspath
```

Comment:
Master build pipeline had failures in past due to fortify, hence it has been isolated into own classpath. Has it been verified to not cause problems anymore, i.e. by running locally or any other means?

Reply:
Couple notes - master build pipeline should not have fortify scan, it is only enabled on nightly master, I guess that is what you mean by master. Second - fortify result is ignored - https://github.com/hmcts/rd-user-profile-api/blob/master/build.gradle#L290, so no matter what fortify execution does, it should not fail pipeline. I saw example where fortify lib was not included and it was still passing. In my understanding, if it builds, then there should be no issues on pipeline. I'll run it on nightly-dev and will report back

Reply:
it is fine on nightly-dev pipeline - https://build.hmcts.net/view/RD/job/HMCTS_j_to_z_Nightly/job/rd-user-profile-api/job/nightly-dev/